core/backend/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -1130,4 +1130,18 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
+  it("biscore update is valid", async () => {

```

Comment:
@MichaelBelousov have you tried skipping this test to see if you can get the backend ones to pass? I haven't been seeing any other PRs with such an ability to reproduce the error you're seeing here.

Reply:
I haven't tried doing that to see if the CI passes, but I will. This test shouldn't be doing that much new compared to other tests we already have here.